app/models/constants.ts

```diff
@@ -177,4 +177,10 @@ export const KWH_M2_YR_NEWBUILDS_RETROFIT = {
   T: 20,
   S: 51,
   D: 55
-}
\ No newline at end of file
+}
+
+export const KG_CO2_PER_KWH = 0.185 // from Carbon Independent https://www.carbonindependent.org/15.html#:~:text=Natural%20gas&text=Older%20gas%20meters%20measure%20gas,kg%20%2F%20kWh%20%5B9%5D%20.
+
+export const NHS_SAVINGS_PER_HEAD_PER_YEAR = 24.78 // from BRE report in 2021, NHS first-year care costs due to poor housing in England individualised
```

Comment:
If you change these to JSDoc comments, IDEs will pick them up which is a nice DX improvement ✨ 

![image](https://github.com/user-attachments/assets/d9868b28-2bd4-459d-a6c6-60411565e4ac)